app/components/HomePage/HomePagePitch/index.js

```diff
@@ -2,31 +2,65 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { FormattedMessage } from 'react-intl'
 import Project from 'static/img/home/project-1.jpg'
-import Event from 'static/img/home/event-1.jpg'
-import Trip from 'static/img/home/trip-1.jpg'
+import Cinnober from 'static/img/home/cinnober.jpg'
+import Tritech from 'static/img/home/tritech.jpg'
+import SanFransisco from 'static/img/home/sanfrancisco.jpg'
+import NewYork from 'static/img/home/newyork.jpg'
 
 import messages from './messages'
 import styles from './styles.css'
 
+const projectNode = () => {
+  return (
+    <div className={styles.images}>
+      <img className={styles.image} src={Project} />
+    </div>
+  )
+}
+
+const eventNode = () => {
+  return (
+    <div className={styles.images}>
+      <div className={styles.eventImageContainer}>
+        <img id={styles.tritech} className={styles.image} src={Tritech} />
+        <img id={styles.cinnober} className={styles.image} src={Cinnober} />
+      </div>
+    </div>
+  )
+}
+
+const tripNode = () => {
+  return (
+    <div className={styles.images}>
+      <div className={styles.tripImageContainer}>
+        <img
+          id={styles.sanFransisco}
+          className={styles.image}
+          src={SanFransisco}
+        />
+        <img id={styles.newYork} className={styles.image} src={NewYork} />
+      </div>
+    </div>
+  )
+}
+
 function HomePagePitch() {
   return (
     <div className={styles.pitch}>
       <h1 className={styles.headline}>
         <FormattedMessage {...messages.headline} />
       </h1>
-      <Section image={Project} {...messages.project} />
-      <Section image={Event} {...messages.events} />
-      <Section image={Trip} {...messages.trip} />
+      <Section {...messages.project}>{projectNode()}</Section>
```

Comment:
I think if we rename `projectNode` to `ProjectNode`, we can instead write:
```js
<Section {...messages.project}>
  <ProjectNode />
</Section>
```
I think that looks a bit nicer maybe?

Reply:
Yeah, it does. Good suggestion.

Reply:
Fixed @willwull